oktoberfest/ce_calibration.py

```diff
@@ -149,11 +161,11 @@ def gen_lib(self, df_search: Optional[pd.DataFrame] = None):
 
     def get_hdf5_path(self) -> str:
         """Get path to hdf5 file."""
-        return self.out_path + ".hdf5"
+        return str(self.out_path) + ".hdf5"
```

Comment:
do we really need to return a string here? Python handles string conversion correctly but you can also return a path with_suffix(".hdf5") if that makes life easier downstream?